src/mscorlib/src/System/Globalization/TextInfo.cs

```diff
@@ -482,6 +482,15 @@ public unsafe String ToTitleCase(String str)
                 charType = CharUnicodeInfo.InternalGetUnicodeCategory(str, i, out charLen);
                 if (Char.CheckLetter(charType))
                 {
+                    // Special case to check for Dutch specific titlecasing with
+                    // "IJ" characters at the beginning of word
+                    if (IsDutchCulture(str) && IsIjAtCurrentPosition(str, i))
```

Comment:
Just updated this by adding the `isDutchCulture` variable outside of the loop, which should address that issue.